packages/bridge-ui-v2/src/components/Bridge/Recipient.svelte

```diff
@@ -0,0 +1,123 @@
+<script lang="ts">
+  import { t } from 'svelte-i18n';
+  import { type Address, isAddress } from 'viem';
+
+  import { Button } from '$components/Button';
+  import { Icon } from '$components/Icon';
+  import { InputBox } from '$components/InputBox';
+  import { Tooltip } from '$components/Tooltip';
+  import { shortenAddress } from '$libs/util/shortenAddress';
+  import { uid } from '$libs/util/uid';
+  import { account } from '$stores/account';
+
+  import { recipientAddress } from './state';
+
+  let dialogId = `dialog-${uid()}`;
+
+  let modalOpen = false;
+  let invalidAddress = false;
```

Comment:
Invalid address is unused

Reply:
It's unused at the moment. I'm waiting for design for this